node-notes-cli/app.js

```diff
@@ -0,0 +1,157 @@
+import { writeFile } from 'node:fs/promises';
+import { readFileSync } from 'node:fs';
+
+// Set up function and variables to handle user inputs
+const action = process.argv[2];
+const arg3 = process.argv[3];
+const arg4 = process.argv[4];
+const legalActions = ['read', 'create', 'update', 'delete'];
+
+function interpretInputs() {
+  switch (action) {
+    case 'read':
+      readNotes();
+      break;
+    case 'create':
+      newNote();
+      break;
+    case 'update':
+      updateNote();
+      break;
+    case 'delete':
+      deleteNote();
+      break;
+    default:
+      console.error(action, 'is not an acceptable action. Please try again with:', legalActions);
+      process.exit(1);
+  }
+}
+
+/**
```

Comment:
I would suggest that you flip the style of comments you have here.

This one (Reusable functions) is more about the code structure, so it's not really a JSDoc comment. I like to do something like:
`// ----------- Reusable functions -----------`
which sets off the comment nicely.

Then the comments above each function _are_ JSDoc comments so use `/**`. You can put it all on one line if you want the code to be more brief:
`/** Catch error handling */`